src/fastapi_poe/base.py

```diff
@@ -612,8 +615,14 @@ def make_prompt_author_role_alternated(
         return new_messages
 
     @staticmethod
-    def text_event(text: str) -> ServerSentEvent:
-        return ServerSentEvent(data=json.dumps({"text": text}), event="text")
+    def text_event(
+        text: str, data: Optional[Dict[str, Any]] = dict()
```

Comment:
let's make this  `data: Optional[MappingProxyType] = None` since we don't want default args and if we're passing this dict around, we prob want it to be read-only? (have ppl copy it like `{**data}` if they need to create a mutable copy)